torchao/experimental/ops/mps/setup.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+from setuptools import setup
+from torch.utils.cpp_extension import CppExtension, BuildExtension
+
+setup(
+    name="torchao_mps_ops",
+    version="1.0",
+    ext_modules=[
+        CppExtension(
+            name="torchao_mps_ops",
+            sources=["register.mm"],
```

Comment:
I think you either need to package your kernels into the wheel, or using `bin2c` just convert them to headers. Or convert them into a metallib